File: backend/User/src/main/java/tn/esprit/user/dto/UserAdminDto.java
package tn.esprit.user.dto;

import lombok.Data;
import tn.esprit.user.entity.UserRole;
import java.time.LocalDateTime;

@Data
public class UserAdminDto {
    private String userId;
    private String name;
    private String email;
    private UserRole role;
    private String phone;
    private boolean isVerified;
    private LocalDateTime createdAt;
    private String profilePicture;
}

File: user-node-service/src/users/dto/user-admin.dto.ts
import { UserRole } from '../schemas/user-role.enum';

export class UserAdminDto {
  userId: string;
  name: string;
  email: string;
  role: UserRole;
  phone?: string;
  isVerified: boolean;
  createdAt: Date;
  profilePicture?: string;
}
